docs/components/base/sensor-controlled.md

```diff
@@ -82,3 +83,7 @@ The following base control API methods are available with feedback control on a
 
 - [SetVelocity()](/components/base/#setvelocity): available if base is configured to receive angular and linear velocity feedback.
 - [Spin()](/components/base/#spin): available if base is configured to receive orientation feedback.
```

Comment:
so spin still will do this as it has been it basically just doesn't use a PID loop. similarly, set velocity will still use sensor feedback but if you don't configure the control parameters it won't use a PID loop. if that makes sense

Reply:
ahh, i see

Reply:
(will read the page again with this context and either re-comment or resolve)

Reply:
Okay, I would suggest the following:

* Adjust the "Requirements" alert box to say something like:

In order to use feedback control for SetVelocity, you must provide a movement sensor that implements GetAngularVelocity and GetLinearVelocity.

In order to use feedback control for Spin, you must provide a movement sensor that implements GetOrientation.

* Remove the following text since it's very similar to what's in the Requirements alert box:

```
The following base control API methods are available with feedback control on a sensor-controlled base:

[SetVelocity()](https://docs-test.viam.dev/2547/components/base/#setvelocity): available if base is configured to receive angular and linear velocity feedback.
[Spin()](https://docs-test.viam.dev/2547/components/base/#spin): available if base is configured to receive orientation feedback.
```

* Maybe rename `Control base velocity with a sensor` to `Feedback control for SetVelocity` and move it above `Configure a sensor-controlled base as follows:`.

* Maybe add a section `Feedback control for Spin` below `Feedback control for SetVelocity`, and have it say something like:

Spin implements a basic form of feedback control that increases or decreases the power being set on the motors in increments of 10%.

Reply:
here I'm also not sure spin is doing the 10% increment. I'll take a closer look at the code and get back to you

Reply:
thank you!

Reply:
okay spin starts the motors at the specified degrees/sec by calling `SetVelocity` at that speed on the underlying base. then it does some math and monitors how far the base has spun and stops once it has reached the goal position

Reply:
this is only if the desired degree < 360. any spin call for more than 360 deg just calls the underlying base's Spin

Reply:
got it, thanks for looking into it! updated the suggestion for `Feedback control for Spin`

Reply:
Ok - made some changes, let me know what you think

Reply:
Looks great!